src/ports/linux/sampleapp_main.c

```diff
@@ -316,6 +316,9 @@ int main (int argc, char * argv[])
    memset (&appdata, 0, sizeof (appdata));
    appdata.alarm_allowed = true;
 
+   /* Enable line buffering for printouts */
```

Comment:
IIRC, line-buffering is the default on Linux (and rt-kernel) if stdout is a TTY. Explicitly setting it might be useful if the output is redirected to a log file (which would otherwise be buffered). If that is the intention then a comment to that effect might be useful.

Reply:
```
/* Enable line buffering for printouts, especially when logging to
      the journal (which is default when running as a systemd job) */
   setvbuf (stdout, NULL, _IOLBF, 0);
```
Done